eftest/src/eftest/runner.clj

```diff
@@ -16,24 +16,44 @@
   (or (-> v meta :eftest/synchronized true?)
       (-> v meta :ns meta :eftest/synchronized true?)))
 
+(defn- known-slow? [v]
+  (or (-> v meta :eftest/slow true?)
+      (-> v meta :ns meta :eftest/slow true?)))
+
 (defn- failed-test? []
   (or (< 0 (:error @test/*report-counters* 0))
       (< 0 (:fail @test/*report-counters* 0))))
 
+(defn- run-test-and-report-time
+  [test-warn-length start-time v]
+  (let [result   (test/test-var v)
+        end-time (System/nanoTime)
+        duration (/ (- end-time start-time) 1e6)]
+    (when (and (not (known-slow? v))
+               (number? test-warn-length)
+               (<= test-warn-length duration))
+      (binding [clojure.test/*testing-vars* [v]]
```

Comment:
What's the purpose of this binding?

Reply:
It's being used to get the necessary information to print out the info on the variable ( https://github.com/tanzoniteblack/eftest/blob/long-test-warnings/eftest/src/eftest/report/pretty.clj#L34 ). My initial reaction was to update that function to take in the variable in question, rather than mucking with the bindings, but I figured if it was being used to get the variable in question here, it's probably being used that way in other reporters, and it would be easier for other people to extend reporters with slow test warnings if I didn't require them to change how they found the variable in question to print information about it.

Reply:
Okay, but let's make it consistent with `clojure.test`. Can you change it to:

```clojure
(binding [*testing-vars* (conj *testing-vars* v)]
```